docs/tutorials/cargo.md

```diff
@@ -0,0 +1,119 @@
+# Setting up your Cargo.toml file for ICU4X
+
+ICU4X makes heavy use of small crates and Cargo features in order to be highly modular. This tutorial is intended to help you set up a Cargo.toml file to download what you need for ICU4X.
+
+## Basic Cargo.toml with testdata
+
+The most basic Cargo.toml to get you off the ground is the following:
+
+```toml
+[package]
+name = "demo"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+icu = "1.0"
+icu_testdata = "1.0"
+```
+
+In your main.rs, you can use most functionality of ICU4X.
+
+[« Fully Working Example »](./cargo_tests/testdata)
+
+## Cargo.toml with experimental testdata
+
+If you wish to use an experimental feature, such as `icu_segmenter`, set up your Cargo.toml like this:
+
+```toml
+[package]
+name = "demo"
+version = "0.1.0"
+edition = "2021"
+publish = false
+
+[dependencies]
+icu = { version = "1.0", features = ["experimental"] }
```

Comment:
Make sure to update this after your other PR

Reply:
Also requires 1.1

Reply:
Done